recipes/tpp/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "TPP" %}
+{% set version = "5.0.0" %}
+{% set sha256 = "caed7a664be42fb9a168df699204816672e640569179cddaa42cf178d339534e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/sashimi/Trans-Proteomic%20Pipeline%20%28TPP%29/TPP%20v5.0%20%28Typhoon%29%20rev%200/TPP_5.0.0-src.tgz
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - tpp5.0.0.patch
+
+build:
+    number: 0
+    skip: true  # [osx]
+    skip: true  # [win]
+    detect_binary_files_with_prefix: true
+
+requirements:
+    build:
+        - gcc <=4.9 # [linux]
+        - perl
+        - perl-xml-parser
+        - zlib
+        - bzip2
+        - perl-extutils-makemaker
+        - perl-app-cpanminus
+    run:
+        - libgcc <=4.9 # [linux]
+        - perl
+        - perl-xml-parser
+        - hardklor
```

Comment:
hardklor is now a dependency? Interesting!

Reply:
It is wanted/needed by one of the TPP programs. Rather than let TPP build it itself, possibly overruling a version installed by conda or elsewhere, it seemed better to just add it as a dependency.

Reply:
Ok!